test/Parse/enum_floating_point_raw_value.swift

```diff
@@ -3,12 +3,11 @@
 // FIXME: this test only passes on platforms which have Float80.
 // <rdar://problem/19508460> Floating point enum raw values are not portable
 
-// REQUIRES: CPU=i386 || CPU=x86_64
```

Comment:
The tests fails on arm platforms due to this change. See FIXME above.

Reply:
Already fixing it in https://github.com/swiftlang/swift/pull/75784. This change as unintentional.